pom.xml

```diff
@@ -23,6 +23,13 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
```

Comment:
I think if you pull from master, this dependency should already be there.

Reply:
Oh, true! I'll do that :)